k8s/helm/nestjs-app/values.yaml

```
@@ -0,0 +1,232 @@
+security:
+  podSecurityContext:
+    fsGroup: 1001
+    runAsNonRoot: true
+  securityContext:
+    runAsUser: 1001
+    runAsGroup: 1001
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: false
+    capabilities:
+      drop:
+      - ALL
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)